components/supervisor-api/terminal.proto

```diff
@@ -44,6 +44,12 @@ service TerminalService {
         };
     }
 
+    rpc GetTaskOutput(GetTaskOutputRequest) returns (GetTaskOutputResponse) {
```

Comment:
Love to see the progress here! :orange_heart: 

There are a couple of things here that are not clear to me right away, maybe it would be great to discuss in sync beforehand:
 - Why can't we (re-)use `Listen`? It almost already does what we want (at least from the outside)
   - Are we seeing a "not found" error, or just no updates?
   - If it's the later, it might be very easy to extend with an additional flag `includeHistory`or sth
   - If it's the former, it might still be the easiest to go a similar route, but make sure that terminals stay around for longer
 - If we really have to introduce a new method:
   - It should be a `stream`
   - it should fit into the existing terminology (task ws terminal) and overall naming pattern

Reply:
I haven't actually thought about integrating that functionality back into `Listen`, and it seems it could actually be easier and not require so many code changes. I'm experimenting with that now

Reply:
I haven't actually thought about integrating that functionality back into `Listen`, and it seems it could actually be easier and not require so many code changes. I'm experimenting with that now